addons/l10n_in_ewaybill/models/l10n_in_ewaybill.py

```diff
@@ -339,6 +340,12 @@ def _check_partners(self):
             error_message += self._l10n_in_validate_partner(partner)
         return error_message
 
+    def _check_state(self):
+        error_message = []
+        if self.account_move_id.state != 'posted':
+            error_message.append(_("An E-way bill cannot be generated for a cancelled or draft invoice."))
```

Comment:
```suggestion
            error_message.append(_("An E-way bill cannot be generated for a %s move", self.state))
```
move is for a generic term. Because it can be invoice/bill/debit note,etc